index.ts

```diff
@@ -0,0 +1,10 @@
+import * as yaml from "yamljs";
+import { GithubExtractor } from "./github-extractor";
+import { PRsProcessor } from "./pr-processor";
```

Comment:
Consider using single quotes for string literals.

Reply:
adsadad

Reply:
No